### ISSUE TITLE
Jump to image link shows error

### ISSUE THREAD

**Issue by octref** [Microsoft/vscode#56158]
Issue Type: <b>Bug</b>

- Have a html file:
  ```html
  <img src="./test.jpg" alt="">
  ```
- Click on the src link (make sure `./test.jpg` is there)
- VS Code shows devtool with errors

Trace:

```
index.js:5 [uncaught exception]: TypeError: Cannot read property 'lineNumber' of null
onError @ index.js:5
index.js:5 TypeError: Cannot read property 'lineNumber' of null
    at Function.e.containsPosition (/Applications/Visual Studio Code - Insiders.app/Contents/Resources/app/out/vs/workbench/workbench.main.js:487)
    at t._getItemEnclosingPosition (/Applications/Visual Studio Code - Insiders.app/Contents/Resources/app/out/vs/workbench/workbench.main.js:1304)
    at t.getItemEnclosingPosition (/Applications/Visual Studio Code - Insiders.app/Contents/Resources/app/out/vs/workbench/workbench.main.js:1304)
    at t.getItemEnclosingPosition (/Applications/Visual Studio Code - Insiders.app/Contents/Resources/app/out/vs/workbench/workbench.main.js:1307)
    at e._getOutlineElements (/Applications/Visual Studio Code - Insiders.app/Contents/Resources/app/out/vs/workbench/workbench.main.js:2322)
    at /Applications/Visual Studio Code - Insiders.app/Contents/Resources/app/out/vs/workbench/workbench.main.js:2322
    at /Applications/Visual Studio Code - Insiders.app/Contents/Resources/app/out/vs/workbench/workbench.main.js:199
onError @ index.js:5
```

VS Code version: Code - Insiders 1.26.0-insider (c827480be25bd63065e7ee2e106c24a707c08004, 2018-08-08T18:46:20.998Z)
OS version: Darwin x64 17.7.0

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i7-4770HQ CPU @ 2.20GHz (8 x 2200)|
|GPU Status|2d_canvas: enabled<br>checker_imaging: disabled_off<br>flash_3d: enabled<br>flash_stage3d: enabled<br>flash_stage3d_baseline: enabled<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>native_gpu_memory_buffers: enabled<br>rasterization: enabled<br>video_decode: enabled<br>video_encode: enabled<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|2, 2, 2|
|Memory (System)|16.00GB (2.05GB free)|
|Process Argv|/Applications/Visual Studio Code - Insiders.app/Contents/MacOS/Electron -psn_0_5305615|
|Screen Reader|no|
|VM|0%|

</details><details><summary>Extensions (13)</summary>

Extension|Author (truncated)|Version
---|---|---
markdown-emoji|bie|0.0.7
markdown-preview-github-styles|bie|0.1.2
tslint|eg2|1.0.34
vsc-material-theme|Equ|2.3.0
prettier-vscode|esb|1.6.1
beautify|Hoo|1.3.2
theme-monokai-pro-vscode|mon|1.1.8
debugger-for-chrome|msj|4.8.2
vetur|oct|0.12.6
polacode|pnp|0.2.2
vscode-sort-json|ric|1.13.0
language-stylus|sys|1.9.2
vim|vsc|0.16.0

(2 theme extensions excluded)

</details>

**Comment by octref** [comment]
This blocks shipping #29899, so I'll move it to August.

**Comment by mjbvz** [comment]
@octref I can't repo this. Do you still see this issue?

**Comment by octref** [comment]
@mjbvz I just merged fix for #29899 yesterday and it's not in insiders yet. Once it arrives, you can repro it.

**Comment by mjbvz** [comment]
@jrieken Here's the error I see in master:

```
index.js:46 TypeError: Cannot read property 'lineNumber' of null
    at Function.Range.containsPosition (range.ts:93)
    at OutlineGroup._getItemEnclosingPosition (outlineModel.ts:159)
    at OutlineGroup.getItemEnclosingPosition (outlineModel.ts:153)
    at OutlineModel.getItemEnclosingPosition (outlineModel.ts:420)
    at EditorBreadcrumbsModel._getOutlineElements (breadcrumbsModel.ts:195)
    at breadcrumbsModel.ts:180
    at async.ts:665
```

This happens because the editor position is null
